projects/swimlane/ngx-charts/src/lib/common/axes/x-axis-ticks.component.ts

```diff
@@ -51,6 +85,10 @@ export class XAxisTicksComponent implements OnChanges, AfterViewInit {
   @Input() gridLineHeight;
   @Input() width;
   @Input() rotateTicks: boolean = true;
+  @Input() markers;
```

Comment:
I saw you defined the markers properties on the sample code. Would you provide such properties inside an Interface for this Input?

Reply:
This is a good one. Created a new model-file for such interfaces as 'Marker' in this context. Strong typing could be improved overall in component in corresponding areas.

Reply:
Yeah, I know, but unfortunately providing strong typing for every component would take some time as most PR are still awaiting to get reviewed and merged.